docs/how-to-guides/feathr-snowflake-guide.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Using Snowflake with Feathr
+parent: Feathr How-to Guides
+---
+
+# Using Snowflake with Feathr
+
+Currently, feathr supports using Snowflake as a source.
+
+# Using Snowflake as a source
+
+To use Snowflake as a source, we need to create a `SnowflakeSource` in projects.
+
+```
+source = feathr.SnowflakeSource(name: str, database: str, schema: str, dbtable: optional[str], query: Optional[str])
+```
+
+* `name` is the source name, same as other sources.
+* `database` is SF database that stores the table of interest
+* `schema` is SF schema that stores the table of interest
```

Comment:
how do users get the schema, could you give an example of schema that users pass in here? thanks

Reply:
yeah snowflake schema is a string value that can be retrieved through UI. storing a table in snowflake requires a database and schema to store the table in. Example (For HBO): in the DEV database there might be a CONTENT schema that will store all of the content related tables

Reply:
Would adding a few examples in the wiki help other users to understand how to use this?

Overall the PR looks great.

Reply:
Sure I will go ahead and add some examples to the docs

Reply:
@hangfei since snowflake schemas are unique to each organization. I have linked the snowflake guide for the same on the wiki